hypertrace-core-graphql-log-event-schema/src/main/java/org/hypertrace/core/graphql/log/event/dao/MockLogEventDao.java

```diff
@@ -0,0 +1,73 @@
+package org.hypertrace.core.graphql.log.event.dao;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.reactivex.rxjava3.core.Single;
+import java.util.List;
+import java.util.Map;
+import lombok.experimental.Accessors;
+import org.hypertrace.core.graphql.log.event.request.LogEventRequest;
+import org.hypertrace.core.graphql.log.event.schema.LogEvent;
+import org.hypertrace.core.graphql.log.event.schema.LogEventResultSet;
+
+public class MockLogEventDao implements LogEventDao {
+
+  @Override
+  public Single<LogEventResultSet> getLogEvents(LogEventRequest request) {
+    ObjectMapper objectMapper = new ObjectMapper();
+    Map<String, String> attributes = Map.of();
+    String attributeString;
+    try {
+      attributeString = objectMapper.writeValueAsString(attributes);
+    } catch (JsonProcessingException e) {
+      attributeString = "";
+    }
+
+    LogEvent logEvent1 =
+        new MockLogEvent(
+            Map.of(
+                "spanId",
+                "span-1",
+                "traceId",
+                "trace-1",
+                "timestamp_nanos",
+                10000L,
+                "attributes",
+                attributeString));
+
+    LogEvent logEvent2 =
+        new MockLogEvent(
+            Map.of(
+                "spanId",
+                "span-2",
+                "traceId",
+                "trace-1",
+                "timestamp_nanos",
+                20000L,
+                "attributes",
+                attributeString));
+    LogEventResultSet logEventResultSet =
+        new MockLogEventResultSet(List.of(logEvent1, logEvent2), 2, 2);
+
+    return Single.just(logEventResultSet);
+  }
+
+  @lombok.Value
+  @Accessors(fluent = true)
+  private static class MockLogEvent implements LogEvent {
+    Map<String, Object> attributeValues;
+
+    @Override
+    public Object attribute(String key) {
+      return this.attributeValues.get(key);
```

Comment:
Attribute will be referencing attribute schema attributes rather than keys in the json string (or is this just here for the sake of mock data)?

Reply:
Yes, Attribute will be referencing attribute schema attributes. json string is just another attribute. 
Added sample request and response in description